src/files/coverageparser.ts

```diff
@@ -1,10 +1,10 @@
-import {parseContent as parseContentClover} from "@cvrg-report/clover-json";
-import {parseContent as parseContentCobertura} from "cobertura-parse";
-import {parseContent as parseContentJacoco} from "@7sean68/jacoco-parse";
-import {Section, source} from "lcov-parse";
-import {OutputChannel} from "vscode";
+import { parseContent as parseContentClover } from "@cvrg-report/clover-json";
+import { parseContent as parseContentCobertura } from "cobertura-parse";
+import { parseContent as parseContentJacoco } from "@7sean68/jacoco-parse";
+import { Section, source } from "lcov-parse";
+import { OutputChannel } from "vscode";
 
-import {CoverageFile, CoverageType} from "./coveragefile";
+import { CoverageFile, CoverageType } from "./coveragefile";
```

Comment:
Yeahhh this file hasn't been touched in a while, didn't even have the latest lintings 😓.